main.go

```diff
@@ -300,14 +301,30 @@ The commands work as following:
 		return
 	}
 
+	tunnelInfoPort, err := arguments.Int("--tunnel-info-port")
+	if err != nil {
+		tunnelInfoPort = tunnelInfoDefaultHttpPort
+	}
+
+	startTunnelCommand, _ := arguments.Bool("start-tunnel")
 	udid, _ := arguments.String("--udid")
 	address, addressErr := arguments.String("--address")
 	rsdPort, rsdErr := arguments.Int("--rsd-port")
 
 	device, err := ios.GetDevice(udid)
-	exitIfError("Device not found: "+udid, err)
-	if addressErr == nil && rsdErr == nil {
-		device = deviceWithRsdProvider(device, udid, address, rsdPort)
+	// device address and rsd port are only available after the tunnel started
+	if !startTunnelCommand {
```

Comment:
@danielpaulus this is the part I don't like when it comes to the `start-tunnel` command (and that name can change as well I guess). We try to get a device, while we are simply start a tunnel for all connected devices, and the parameters for `--address` and `--rsd-port` are usable for all other commands, apart from `start-tunnel`